ion_phys/rate_equations.py

```diff
@@ -72,7 +73,8 @@ def get_stim(self, lasers):
                 for laser in [laser for laser in _lasers if laser.q == q]:
                     delta = delta_lu - laser.delta
                     I = laser.I
-                    R += GammaJ2/(4*np.power(delta, 2) + GammaJ2)*I*(Q*Gamma)
+                    R += GammaJ2/(4*np.power(delta, 2) + GammaJ2)
+                    R *= I*(Q*Gamma_reduced)
```

Comment:
why move this onto a new line?

Reply:
It's a long line. Broke it up for flake8. Messed up the maths though, will fix.